src/core/controller/account/accountLoginClicked.ts

```
@@ -25,6 +26,8 @@ export async function accountLoginClicked(controller: Controller): Promise<Strin
 	const authUrl = vscode.Uri.parse(
 		`https://app.cline.bot/auth?state=${encodeURIComponent(nonce)}&callback_url=${encodeURIComponent(`${uriScheme || "vscode"}://saoudrizwan.claude-dev/auth`)}`,
 	)
-	vscode.env.openExternal(authUrl)
-	return authUrl.toString()
+	await vscode.env.openExternal(authUrl)
```

How does the await benefit here?

---

Without it, if openExternal throws an exception the error is lost.